js/foundation.util.mediaQuery.js

```diff
@@ -173,7 +178,7 @@ var MediaQuery = {
    * @private
    */
   _watcher() {
-    $(window).on('resize.zf.mediaquery', () => {
+    $(window).off('resize.zf.mediaquery').on('resize.zf.mediaquery', () => {
```

Comment:
sure... `.zf.mediaquery` is putting a namespace on this so that we can target it specifically, and we run off and on to make this idempotent (ie we only ever have one listener, not a new one for every time this fn is called)